src/app/components/MainMenu.js

```diff
@@ -0,0 +1,47 @@
+export class MainMenu {
+  constructor(id, content, gameModeIndex) {
+    this.gameModeIndex = gameModeIndex,
+      this.render(id, content);
+  }
+
+  renderContent(id, content) {
+    const mainMenuContainer = document.querySelector(`#${id}`);
+    mainMenuContainer.classList.add('mainMenu');
+    for (let i = 0; i < 3; i++) {
+      mainMenuContainer.appendChild(document.createElement('div'));
+    }
```

Comment:
Spróbujmy zastąpić tego for, zamiast tego użyjmy map, osobiście jak juz bedziemy umieli korzystać z map to for juz sie praktycznie nie używa

Reply:
Maciek, ale tu nie ma po czym mapować, mainMenuContainer nie jest tablicą tylko jednym elementem, w którym tworzę trzy inne divy, ewentualnie można tu zrobić innerHtml, ale w założeniach projektu ustaliliśmy, że mają być appendChildy... no  chyba, że źle rozumiem Twój komentarz

Reply:
Hmmm... Powinniśmy mapować po naszym content, jeżeli w przyszłości byśmy chcieli rozszerzyć ilość kategorii na 4 to w takim wypadku byśmy musieli znaleźć tego fora i zmienić jego wartość, aczkolwiek masz rację, że tutaj może nie bedziemy wewnatrz tej metody korzystać z tego co znajduję się bezpośrednio w content, a więc najlepszą alternatywną opcją będzie tutaj zamienić "3" na "content.lenght"

Reply:
jasne, gotowe,